enterprise/frontend/src/embedding-sdk/components/public/CollectionBrowser/CollectionBrowser.tsx

```diff
@@ -23,6 +23,19 @@ const USER_FACING_ENTITY_NAMES = [
 
 type UserFacingEntityName = (typeof USER_FACING_ENTITY_NAMES)[number];
 
+type CollectionBrowserListColumns =
+  | "type"
+  | "name"
+  | "lastEditedBy"
+  | "lastEditedAt";
+
+const COLLECTION_BROWSER_LIST_COLUMNS: CollectionBrowserListColumns[] = [
+  "type",
+  "name",
+  "lastEditedBy",
+  "lastEditedAt",
+];
```

Comment:
tip: you could also do the reverse to generate the type from the array. I noticed that the above "UserFacingEntityName" type already does this exact thing.

```suggestion
const COLLECTION_BROWSER_LIST_COLUMNS= [
  "type",
  "name",
  "lastEditedBy",
  "lastEditedAt",
] as const;

type CollectionBrowserListColumns = (typeof COLLECTION_BROWSER_LIST_COLUMNS)[number]
```